app.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const topics = require('./routers/topics.router');
+
+const app = express();
+
+app.use('/api/topics', topics);
+
+//Error-handler
```

Comment:
Error handling needs to be updated - at the moment this is a partially complete custom error handler - you need to think about what will be sent back i.e. res.status(err.status).send(err.message) or similar. This will only handle custom errors - you need to think about how to handle 404 and 500 errors too.